tiledb/sm/misc/constants.h

```diff
@@ -43,6 +43,15 @@ enum class Compressor : char;
 
 namespace constants {
 
+/**
+ * If `true`, this will check for coordinate duplicates upon sparse
+ * writes.
+ */
+extern bool check_coord_dups;
```

Comment:
Any reason not to be `const`?

Reply:
Nope, they need to be const. Good catch.